vitest.browser.base.config.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { defineConfig } from "vitest/config";
```

Comment:
I know we have a few configs floating around now, how come we needed another?

Reply:
This is without the test-recorder which most of the core packages do not need.  By default, the base config doesn't have it, but the shared one does.